internals/testing/test-bundler.js

```diff
@@ -49,3 +49,5 @@ global.URL.revokeObjectURL = jest.fn();
 
 const noop = () => {};
 Object.defineProperty(global.window, 'scrollTo', { value: noop, writable: true });
+
+global.window.HTMLElement.prototype.scrollIntoView = jest.fn();
```

Comment:
I've added it because the test for <IncidentSplitForm /> directly hooks into <IncidentSplitFormIncident />, clicking the 'Extra deelmelding toevoegen' button and thereby triggering the scrollIntoView effect. 

I figured it would pollute the <IncidentSplitForm /> test to mock scrollIntoView there.

Reply:
Not sure I understand.

Reply:
Discussed with @aditudorache, I'll move the mock to the `<IncidentSplitForm />` test. If the issue reoccurs, we'll reconsider adding it to the `test-bundler.js` file